services/QuillLMS/app/models/app_setting.rb

```diff
@@ -0,0 +1,65 @@
+# == Schema Information
+#
+# Table name: app_settings
+#
+#  id                 :integer          not null, primary key
+#  enabled            :boolean          default(FALSE), not null
+#  enabled_for_admins :boolean          default(FALSE), not null
+#  name               :string           not null
+#  pct_active         :integer          default(0), not null
+#  user_whitelist     :string           default([]), not null, is an Array
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#
+# Indexes
+#
+#  index_app_settings_on_name  (name)
+#
+class AppSetting < ActiveRecord::Base
+    PCT_ACTIVE_VALUES = [100, 90, 80, 70, 60, 50, 40, 30, 20, 10, 0] 
+
+    validates :pct_active, inclusion: { in: PCT_ACTIVE_VALUES }
+    validates :name, uniqueness: true
+
+    def self.enabled?(name:, user:)
```

Comment:
I have open questions about this method. It's not used in the code (and I think it has a bug in it anyway). There is a comment that this method isn't being used in the controller for performance reasons, but if this isn't performant for our use case, maybe there should be a class method that is, e.g. `self.all_enabled_for_user?(user)`?

Reply:
As discussed, we will create another class method to return all of the features that are enabled for the user. This will be used by the index controller endpoint.